.github/workflows/ci.yaml

```diff
@@ -39,6 +51,44 @@ jobs:
 
       - run: go build -o temporal-features
 
+      - name: Get the latest release version
+        id: latest_version
+        run: |
+          go_latest="${{ github.event.inputs.go_sdk_version }}"
+          if [ -z "$go_latest" ]; then
```

Comment:
This script is long enough that I'm tempted to say it ought to go in a file, but, I can see how that's _really annoying_ with the amount of variable interpolation going on. So, up to you if you think that's worth it.

I really wish GH's syntax highlighter at least did a better job here.